src/api/forms/form-metadata-repository.js

```diff
@@ -35,10 +35,9 @@ export async function list() {
  * @param {string} formId - ID of the form
  * @returns {Promise<FormConfiguration>} - form configuration
  */
-export async function get(formId) {
-  const formMetadataFilename = getFormMetadataFilename(formId)
-  const value = await readFile(formMetadataFilename, { encoding: 'utf8' })
-  return JSON.parse(value)
+export function get(formId) {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return -- Allow JSON 'any' type
+  return readFile(getFormMetadataFilename(formId), 'utf-8').then(JSON.parse)
```

Comment:
the corresponding test will need updating to account for the options change

Reply:
Thanks, yeah I missed this one but the test caught it

Hope you don't mind it all on one line for `// eslint-disable-next-line` like we do elsewhere?

https://github.com/DEFRA/forms-manager/blob/002bac49c8a0d16b8598995dbb4cedbeaacdd1cb/src/api/forms/form-definition-repository.js#L39